hyper/client/aws/helpers.go

```diff
@@ -0,0 +1,93 @@
+package aws
```

Comment:
Same on this file.

Reply:
So for this, I just moved a bunch of these functions that we need for making requests but don't have any client logic in them into this file to make the main ec2 client file a bit easier to read through